doc/source/basics.rst

```diff
@@ -1738,19 +1738,26 @@ description.
 Sorting
 -------
 
-There are two obvious kinds of sorting that you may be interested in: sorting
-by label and sorting by actual values.
+Pandas supports three kinds of sorting: sorting by index levels,
```

Comment:
Maybe this isn't correct. But I was trying to be consistent in using `level` to refer to the names of the indexes or columns (`df.index.names`, `df.columns.names`) and `label` to refer to the values referenced by the levels (`df.index.get_level_values(lv1)`, `df.columns.get_level_values(lv1)`). Is that in line with accepted vocabulary? @jorisvandenbossche

Reply:
Personally I think "index level" is the equivalent of "column", and from there it would follow that the equivalent of "column values" is "index labels of one level / index level labels / index level values" (but none of those sound that good) and the equivalent of "column names/labels" would then be "index level name". 
But it's not that we do that very consistently currently, or that this is necessarily agreed upon vocabulary. But I think it is in line more or less with what you say?

The main reason I commented it here, is because the next line uses "column values", and as you say "label to refer to the values" then to be consistent it would be "index labels". 
And a second reason I commented is that I think "index level" is more complicated, as then a user needs to be familiar with the concept of MultiIndexes, while for this section about sorting, that is not needed.

Reply:
Thanks for clarifying.  After looking at it again, I do agree that "index labels" is the better parallel to "columns values".